src/main.ts

```diff
@@ -48,7 +56,7 @@ async function run(): Promise<void> {
     owner,
     repo,
     tag_name: rawTag,
-    name,
+    title,
```

Comment:
> now I am getting a strange result, where the title shown for the release is `tag: subject` (e.g. `v1.2.3: this is text`.
> 
> What is even stranger though is when I go to manually edit the release, the title field is shown as empty (in edit mode)

The release title is not in the request body when it's posted. GH defaults to that when the title is excluded AFAIK.

The issue is here.
```typescript
const foo = makeFoo();

// In JS/TS, these are equivalent:
{ foo }

{ foo: foo }
```

[The API expects a value in the body called `name`](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release), but this change sends `title` instead.

Reply:
Thanks, TypeScript, for catching this invalid object (sarcasm) :laughing:

Reply:
Ah, see there is my great JS experience. Thanks for the catch. Hope my fix was correct. If you don't mind re-building it.

Reply:
https://github.com/spenserblack/actions-tag-to-release/pull/114/commits/f99d1d3709e207a602c00b01980bc78bb123079a